### PR TITLE
Update apps_update.yml

### PR DESCRIPTION
**Pull request by dbrian57**
Updates App update cURL example to be complete.

### CODE REVIEW

File: specification/resources/apps/examples/curl/apps_update.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
-lang: cURL
-source: |-
-  curl -X PUT \
-    -H "Content-Type: application/json" \
-    -H "Authorization: Bearer $DIGITALOCEAN_TOKEN" \
-    "https://api.digitalocean.com/v2/apps/{id}"
+curl -X PUT \
```
Comment (andrewsomething) [inline specification/resources/apps/examples/curl/apps_update.yml:1]:
```suggestion
lang: cURL
source: |-
  curl -X PUT \
```

This is a yaml file. The example needs to be under the `source` key.

**Review by andrewsomething — CHANGES_REQUESTED** [review]
Good catch!

Let a comment inline that should resolve the CI failure. Beyond that, I wonder if we could use a more minimal example? This is pretty hard to read.

File: specification/resources/apps/examples/curl/apps_update.yml (hunk 1 above)
Comment (dbrian57) [reply specification/resources/apps/examples/curl/apps_update.yml:1]: ah, got it. Thanks!

**Comment by dbrian57** [comment]
@andrewsomething sure thing! I work something up.

**Comment by dbrian57** [comment]
@andrewsomething How do you feel about this? Should I uncompress it? I compressed it for easier copying, but maybe that's not the move here

**Review by andrewsomething — APPROVED** [review]
:+1:  Works for me. Thanks Dan!

**Pull request merged**